core/src/perspectives/PerspectiveProxy.ts

```diff
@@ -388,6 +388,25 @@ export class PerspectiveProxy {
         return this.getSubjectProxy(exprAddr, subjectClass)
     }
 
+    /** Removes a subject instance by running its (SDNA defined) constructor,
+     * which means removing links around the given expression address
+     * 
+     * @param subjectClass Either a string with the name of the subject class, or an object
+     * with the properties of the subject class. In the latter case, the first subject class
+     * that matches the given properties will be used.
+     * @param exprAddr The address of the expression to be turned into a subject instance
+     */
+    async removeSubject<T>(subjectClass: T, exprAddr: string) {
+        let className = await this.stringOrTemplateObjectToSubjectClass(subjectClass)
+        let result = await this.infer(`subject_class("${className}", C), class_remover(C, Actions)`)
```

Comment:
Let's call it `destructor`
```suggestion
        let result = await this.infer(`subject_class("${className}", C), destructor(C, Actions)`)
```